src/main/java/dev/sora/relay/game/world/leveldb/LevelDBWorld.kt

```diff
@@ -0,0 +1,78 @@
+package dev.sora.relay.game.world.leveldb
+
+import dev.sora.relay.game.world.chunk.Chunk
+import dev.sora.relay.game.world.chunk.ChunkSection
+import io.netty.buffer.ByteBufAllocator
+import org.iq80.leveldb.CompressionType
+import org.iq80.leveldb.Options
+import org.iq80.leveldb.impl.Iq80DBFactory
+import java.io.File
+
+/**
+ * From PowerNukkitX
+ * @author Superrice666, NekoRibbon
+ */
+class LevelDBWorld(val folder: File) {
+
+	private val db = Iq80DBFactory.factory.open(folder, Options().createIfMissing(true).compressionType(CompressionType.ZLIB_RAW))
```

Comment:
The compression seems not working as expected

Reply:
idk how 2 make the compression working ;w;